logger/logger.go

```diff
@@ -0,0 +1,46 @@
+package logger
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/rifflock/lfshook"
+	"github.com/sirupsen/logrus"
+)
+
+var Log *logrus.Logger
+
+func NewLogger() *logrus.Logger {
+	if Log != nil {
+		return Log
+	}
+	var flag string
+	var filePath string
+	fmt.Println("Enter Path to log debug files: (Y/N)")
```

Comment:
Don't do interactive input like this, it makes it very hard to use your tool in scripts.
Also, this is a logger class, it has no business dealing with the user directly. Follow the principles of Dependency injection. The Caller should pass whatever input is required. In this case the orchestrator should take whatever input is required from the user and initialize the logger accordingly.

Reply:
Okay, At first I was trying this only but then I thought If we have to do this whenever we initialize logger so why not include this in its initialization.
But if this is bad practice i'll try to do other way.